### ISSUE TITLE
Odd behaviour when explicitly installing a package while upgrading the system

### ISSUE THREAD

**Issue by JustHev** [archlinuxfr/yaourt#138]
When I run `yaourt -Syua new-package` (replace new-package with any package not previously installed on the system, in my case, fortune-mod) the following output appears:

    ==> New package :
    community/fortune-mod        1.99.1-6  (required by -)

Also, when pressing [V] to show package details, it doesn't show the description for the explicitly installed package. This behaviour seems wrong, because when explicitly installing a package, the package isn't required by any package and if I want to see details, I should see them. Please fix.

**tl;dr** When explicitly installing a packing while installing updates, a line with (required by -) is shown next to the new package and when requesting details, you don't get details for that new package

**Comment by f2404** [comment]
Does anyone see a reason why a new package should be treated as another package's dependency when installed together with the '-u' option?
The code is written this way (i.e. it's not a bug but rather a feature) but I cannot tell why it's needed. Opinions?

**Comment by f2404** [comment]
@JustHev How important is to run both upgrade and install in one command for you?

**Comment by JustHev** [comment]
@f2404 Not really important, but it is a feature provided by Pacman. The output just seemed slightly odd. I just expected not to see a "required by"-annotation when explicitly installing a package.

**Comment by f2404** [comment]
@JustHev Yes, it's supported by pacman, but it's also supported by yaourt - yaourt will install the package but with some weird output.
The reason I'm discussing here is that as I see it, the guys that wrote that code were trying really hard to make this installed package be treated as a dependency. Though I don't think that such behavior is documented anywhere....

**Comment by larchunix** [comment]
My 2 cents:

I agree with @f2404, this was probably done this way on purpose by the developers.

**Comment by f2404** [comment]
Should we close the issue then?
